kernel_hardening_checker/checks.py

```diff
@@ -51,8 +52,9 @@ def add_kconfig_checks(l, arch):
     l += [OR(KconfigCheck('self_protection', 'defconfig', 'STRICT_MODULE_RWX', 'y'),
              KconfigCheck('self_protection', 'defconfig', 'DEBUG_SET_MODULE_RONX', 'y'),
              modules_not_set)] # DEBUG_SET_MODULE_RONX was before v4.11
-    l += [OR(KconfigCheck('self_protection', 'defconfig', 'REFCOUNT_FULL', 'y'),
-             VersionCheck((5, 5)))] # REFCOUNT_FULL is enabled by default since v5.5
+    if arch in ('X86_64'):
```

Comment:
Could you describe the rationale behind this change?

I think `REFCOUNT_FULL` was available for other architectures, not only for `X86_64`.